Lib/json/__init__.py

```diff
@@ -342,6 +344,13 @@ def loads(s, *, encoding=None, cls=None, object_hook=None, parse_float=None,
                             f'not {s.__class__.__name__}')
         s = s.decode(detect_encoding(s), 'surrogatepass')
 
+    if encoding is not None:
+        warnings.warn(
+            "`loads(encoding=..)` keyword argument is deprecated since Python 3.1"
```

Comment:
I don't remember any other case in which the syntax like ``` `loads(encoding=..)` ``` is used. Please write the message in the style conforming to styles of other deprecation messages.

Reply:
Done.